test/cli/detect_breakends_cli_test.cpp

```diff
@@ -52,7 +51,7 @@ TEST_F(cli_test, with_arguments)
         "BND: Reference\tchr22\t17458417\tForward\tReference\tchr21\t41972615\tForward\tm41327/11677/CCS\n"
         "BND: Reference\tchr22\t17458418\tForward\tReference\tchr21\t41972616\tForward\tm21263/13017/CCS\n"
         "BND: Reference\tchr22\t17458418\tForward\tReference\tchr21\t41972616\tForward\tm38637/7161/CCS\n"
-        "Done. Found 5 junctions.\n"
+        "Done. Found 4 junctions.\n"
```

Comment:
Why did the test result change?

Reply:
because of my simple clustering method. Which clustered two junctions into one :)